docs/migrating/from-gatsby.md

```diff
@@ -169,7 +169,7 @@ export default {
 
 ## Search Engine Optimization
 
-Most Gatsby examples use `react-helmet` to assist with adding `meta` tags for proper SEO. With Next.js, we use [`next/head`](/docs/api-reference/next/head.md) to add `meta` tags to your `<head />` element. For example, here's an SEO component with Gatsby:
+Most Gatsby examples use `react-helmet` to assist with adding `meta` tags for proper SEO. With Next.js, we use [`next/head`](/docs/api-reference/next/head.md) to add `meta` tags to your `<head />` element. For example, here's how a SEO component with Gatsby looks like:
```

Comment:
Personally, I think the sentence reads better before. Saying "how" something looks like doesn't make grammatical sense.

Reply:
Really? I use this a lot in docs and I've always thought that's the way.

Reply:
It should either be:

- For example, here's what an SEO component with Gatsby looks like:
- For example, here's an SEO component with Gatsby:

> If you pronounce a letter as a letter and it begins with a vowel sound, you should precede it with an. The consonants with vowel sounds include f, h, l, m, n, r, s, and x.

Source: https://www.grammar.com/a-vs-an-when-to-use/

Reply:
Wait, why `an`?

Reply:
Did you read the link?

Reply:
![image](https://user-images.githubusercontent.com/9113740/95124946-d986d080-0719-11eb-9054-848d3ad2fa54.png)

Reply:
👍 on "an SEO"

Reply:
Acknowledged, I didn't know all the rules 😓